app/models/contact.rb

```diff
@@ -66,6 +64,14 @@ def validate_last_name?
     !profile?
   end
 
+  def full_name_changed?
+    first_name_changed? || last_name_changed?
```

Comment:
`saved_change_to_first_name? || saved_change_to_last_name?`

Reply:
if my callback is before_save?

Reply:
Then maybe `will_save_change_to_attribute?`

Reply:
Ah, now I realize, it is depricated, yea. But my reaction shows that the Rails team did a extremly bad job with explicity and naming there.